proto/buf/alpha/registry/v1alpha1/github.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+message GithubAppConfig {
+    string app_id = 1;
+}
+
+message GetGithubAppConfigRequest{
+    string id = 1;
```

Comment:
I'm guessing `id` is not a good name, but not sure what this ID represents.

Reply:
For now we can leave it blank as it should only have one entry per deployment.

Reply:
OK, meaning just an empty request object?